scrape.sh

```diff
@@ -0,0 +1,22 @@
+#if the .env file exists, then export the environment variables from it
+#.env file sample:
+#=============================================
+#APPLICATION_ID=6TZGXYS27N
+#API_KEY=51151f6285b830cea1d7e40675f8af8f
```

Comment:
Make sure these aren't real.  If they are, delete them now (you shouldn't leak keys into github)